src/accelerate/utils/dataclasses.py

```diff
@@ -585,6 +585,9 @@ class GradientAccumulationPlugin(KwargsHandler):
             `True` if the used scheduler was not adjusted for gradient accumulation.
         sync_with_dataloader (`bool`, *optional*, defaults to `True`):
             Whether to synchronize setting the gradients when at the end of the dataloader.
+        sync_each_batch (`bool`, *optional*, defaults to `False`):
```

Comment:
`False` defaults don't need to be mentioned :) 

```suggestion
        sync_each_batch (`bool`, *optional*):
```